zone/gm_commands/appearanceeffects.cpp

```diff
@@ -0,0 +1,44 @@
+#include "../client.h"
+
+void command_appearanceeffects(Client *c, const Seperator *sep)
+{
+	if (sep->arg[1][0] == '\0' || !strcasecmp(sep->arg[1], "help")) {
+		c->Message(Chat::White, "Syntax: #appearanceeffects [subcommand].");
+		c->Message(Chat::White, "[view] Display all appearance effects saved to your target. #appearanceffects view");
+		c->Message(Chat::White, "[set] Set an appearance effects saved to your target. #appearanceffects set [app_effectid] [slotid]");
+		c->Message(Chat::White, "[remove] Remove all appearance effects saved to your target. #appearanceffects remove");
+	}
+
```

Comment:
Help is not listed as a sub command, also could just initialize target after the initial check if every sub command uses the target. Take a look at all the command stuff I did for a good reference.

Reply:
This is written as intended. You can type nothing or type help and it will provide you with the commands description.
I don't see the value of initializing the target unless its actually going to be used in the correct command.